tests/metrics/test_f1.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python3
+# coding: utf-8
+
+import numpy as np
+import pytest
+import tensorflow as tf
+
+from wellcomeml.metrics import f1_loss, f1_metric
+
+
+@pytest.fixture(scope="module")
+def tmpdir(tmpdir_factory):
+    return tmpdir_factory.mktemp("test_f1")
+
+
+@pytest.fixture(scope="module")
+def data():
+    X_train = np.random.random((100, 10))
```

Comment:
This is a great test fixture. I would say it's not reproducible because of a random seed, but given the type of assertion below, I think it is fine.

Reply:
Yes that's true, i think as you say, we are probably ok with the simple assertion I make for now. Perhaps there will need to be a time when we make a more concrete assertion and we could adjust the fixture.